### DISCUSSION TITLE
Grunt

### DISCUSSION THREAD

**Discussion by D0nnaz** [Sensinki/Blok-Tech-Team-4A#36] [General]
Grunt kan automatisch images, css een js verkleinen. Daarvoor moet je **grunt** typen in je terminal, maar je moet het nog wel installeren. Dat kan je doen met `npm install -g grunt-cli`. Grunt is eig een soort nodemon. Als je grunt typt en je maakt veranderingen dan worden die automatisch geminified.

**Comment by Sensinki** [comment]
ik heb gevoel van dat de website wel sneller is ahahhs
